python/versions.bzl

```diff
@@ -497,6 +497,19 @@ TOOL_VERSIONS = {
         },
         "strip_prefix": "python",
     },
+    "3.13.0": {
```

Comment:
Please update the rest of the toolchains to use the same indygreg release. If the same version is already present in the `TOOL_VERSIONS`, then we should overwrite the old values with the new values.

Reply:
@aignas you mean to update for all other python versions like 3.12.X etc ?
I wonder whether there is an automatic way to get sha256 hashes for the versions? For 3.13 I did that manually, downloading tar.gz and running `sha256sum`...